expo/tests/gbe/test_review_volunteer.py

```diff
@@ -36,29 +36,3 @@ def test_review_volunteer_all_well(self):
         response = review_volunteer(request, volunteer.pk)
         nt.assert_equal(response.status_code, 200)
         nt.assert_true('Bid Information' in response.content)
-
-    def only_show_relevant_events(self):
```

Comment:
1 - it never ran - if you go to master, and remove this test, you'll see it doesn't change the test count at all when you run all the tests.
2 - it's unrunnable.  when I changed the function name form "only_show_relevant_events" to something that begins with the word "test_" it will run, and then gets lots of errors.  I don't think this test ever ran successfully - it was never executed at all so the errors in it where never obvious.
3 - when I refactored, it was obsolete.  The same test is now run in the assign volunteers testing, and now it works.